tests/unittests/Foundation/NSIndexSetTests.mm

```diff
@@ -47,33 +49,51 @@ void verifyIndexSetEqualsArray(NSArray* expected, NSIndexSet* actual) {
     // add in a left overlap.
     LOG_INFO("Left overlap addIndexesInRange operation tests of NSIndexSet");
     [indexSet addIndexesInRange:NSMakeRange(1, 2)]; // [1-2]
-    expected = @[ @1, @2, @3, @4, @5, @7, @8, @11, @12, @13, @14, @15 ];
-    verifyIndexSetEqualsArray(expected, indexSet);
+    // The resulting set should contain ranges - [1-5], [7-8] and [11-15].
+    // verify the ranges in the set.
+    [expectedIndexSet removeAllIndexes];
+    [expectedIndexSet addIndexesInRange:NSMakeRange(1, 5)]; // [1-5]
+    [expectedIndexSet addIndexesInRange:NSMakeRange(7, 2)]; // [7-8]
+    [expectedIndexSet addIndexesInRange:NSMakeRange(11, 5)]; // [11-15]
```

Comment:
nit: this would be neater with a helper function that takes a list/array/va_list of NSRanges.